test/functional/specs/view/dataElements/variable.spec.mjs

```diff
@@ -57,6 +57,18 @@ test.requestHooks(
   await schemaField.expectText("Test Schema 1");
 });
 
+test.requestHooks(sandboxMocks.multipleWithoutDefault)(
+  "initializes the form when no default sandbox is set",
+  async () => {
+    await extensionViewController.init({
+      settings: null,
+    });
+
+    await t.wait(1000);
```

Comment:
The error in page appears async so I need to wait a little in the test to make sure it is not showed.